pkg/utils/git/workspace.go

```diff
@@ -17,5 +18,7 @@ func Modified() ([]string, error) {
 			result = append(result, cols[1])
 		}
 	}
+
+	fmt.Printf(" res %s    result %s \n", res, result)
```

Comment:
@dustinfaulkner91 This is the other extra debug statement that should be removed.